app/src/main/res/layout/fragment_swipeable_pets.xml

```diff
@@ -1,7 +1,8 @@
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This was a key part of the change - use a framelayout to control the z-ordering.